src/App.js

```diff
@@ -1,25 +1,75 @@
-import logo from './logo.svg';
-import './App.css';
+import { useEffect, useState } from "react"
+import Confetti from "react-confetti"
+import Die from "./Die"
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export default function App() {
+    const [dice, setDice] = useState(allNewDice())
```

Comment:
Instead of passing the function into the useState hook, can you use setDice and you pass the newArray returned inside the allNewDice function, afterAll that's is why we need to have the setState function. This will improve the readability of the code.